File: src/codes/Users_Manual/sampling_config/sampling_config_freq_nearest.rs
# use std::time::Duration;
# use autd3::prelude::*;
# fn main() {
# let _ = 
SamplingConfig::new(4000.0 * Hz).into_nearest();
// or
# let _ = 
SamplingConfig::new(Duration::from_micros(250)).into_nearest();
# }

File: src/codes/Users_Manual/sampling_config/sampling_config_freq_nearest.cpp
//~#include<autd3.hpp>
//~#include<chrono>
//~int main() {
//~using namespace autd3;
//~const auto s =
SamplingConfig(4000.0f * Hz).into_nearest();
// or
//~const auto sp =
SamplingConfig(std::chrono::microseconds(250)).into_nearest();
//~return 0; }